engines/bastion_katello/app/assets/javascripts/bastion_katello/content-views/details/filters/views/new-filter.html

```diff
@@ -19,12 +19,9 @@ <h3 translate>Add New Filter</h3>
     <select id="type"
             name="type"
             ng-model="filter.type"
+            ng-options="filterChoice.id as filterChoice.name for filterChoice in filterChoices"
```

Comment:
Does it make sense to use a select here since we only have one type?  I think this should be a hidden field and then become a select if we're planning on adding additional filter types in the future.

You could just add an `ng-show`/`ng-if` to only show the select if `filterChoices.length > 1`.